blockchain/src/multichain-feed/email-notifications/sendNotifications.js

```diff
@@ -154,8 +175,10 @@ const absolutePath = process.cwd() + "/" + path;
       if (error.name === "ExpiredTokenException") {
         token = createJWT(secret, "notification-watcher", algorithm);
         log.info("New JWT token created due to expiration.");
+      } else if (error.name === "InvalidAlgorithmException") {
+        log.error(error, `Notification e-mail request signed with invalid algorithm: ${algorithm}`);
       } else {
-        log.error("Error during notification processing:", error);
+        log.error(error, "Error during notification processing: ", error);
```

Comment:
@SamuelPull is there a reason for logging `error` twice?

Reply:
no